unit/instance.py

```diff
@@ -38,12 +38,25 @@ def __init__(self, console_sock, cluster_cookie):
         self.process = None
         self.sock = socket.socket(socket.AF_UNIX, socket.SOCK_STREAM)
 
+        self.__original_exists = os.path.exists
```

Comment:
It's not clear now why do you save original functions and then restore them. Looks that this functions should be mocked only in tests that uses them, or here in `__init__` function. I think second approach is better, because generally we use role tasks on remote host and there is no cases when we check existence of files on local machine.

Reply:
I restore the original functions because they change not only for the current test, but also for all next tests. This may lead to the fact that in some tests the original function should be used, however, it will use the mocked function.

Also, this functions mocked is `start` function, because it was originally there. But if you want, I'll transfer it to the `__init__` function